about.html

```diff
@@ -13,58 +13,152 @@
 
```

Comment:
How would you need to change your structure / margins / padding / content to match the design file 100% --> look closely at where the boarder edges end on the design file for a hint.